app/table/columns/common.tsx

```diff
@@ -6,21 +6,26 @@
  * Copyright Oxide Computer Company
  */
 
+import { formatISO } from 'date-fns'
 import { format } from 'date-fns/format'
 import { filesize } from 'filesize'
 
 import { Truncate } from '~/ui/lib/Truncate'
 
 import { EmptyCell } from '../cells/EmptyCell'
-import { TwoLineCell } from '../cells/TwoLineCell'
 
 // the full type of the info arg is CellContext<Row, Item> from RT, but in these
 // cells we only care about the return value of getValue
 type Info<T> = { getValue: () => T }
 
 function dateCell(info: Info<Date>) {
   const date = info.getValue()
-  return <TwoLineCell value={[format(date, 'MMM d, yyyy'), format(date, 'p')]} />
+  return (
+    <time dateTime={formatISO(date)} className="flex flex-wrap gap-x-2">
```

Comment:
`date.toISOString()` should cover this. Not sure why they bothered reimplementing the whole thing themselves. I guess it takes options that let you do date only or time only. 

https://github.com/date-fns/date-fns/blob/5c1adb5369805ff552737bf8017dbe07f559b0c6/src/formatISO/index.ts#L47

Reply:
Oh, good point. Merp.